### PR TITLE
tvOS Reference Docs Changes

### PR DESCRIPTION
**Pull request by DellaBitta**
Update the reference docs to refer to tvOS when discussing particular methods and classes.

***

[Packaging Build for docs push](https://github.com/firebase/firebase-cpp-sdk/actions/runs/3750654704)
***

### Type of Change
Place an `x` the applicable box:
- [ ] Bug fix. Add the issue # below if applicable.
- [ ] New feature. A non-breaking change which adds functionality.
- [X] Other, such as a build process or documentation change.
***

### CODE REVIEW

File: auth/src/include/firebase/auth.h [hunk 1]
```diff
@@ -174,7 +174,7 @@ class Auth {
   void set_language_code(const char* language_code);
 
   /// Sets the user-facing language code to be the default app language. This
-  /// uses a languge associated with the phone's locale data.  On desktop
+  /// uses a languge associated with the device's locale data.  On desktop
```
Comment (jonsimantov) [inline auth/src/include/firebase/auth.h:177]: While you're here, can you fix the typo "languge" => "language"?

File: firestore/src/include/firebase/firestore/settings.h [hunk 2]
```diff
@@ -168,7 +168,7 @@ class Settings final {
    * `set_dispatch_queue()`.
    *
    * @note This method is only available when `__OBJC__` is defined, such as
-   * when compiling for iOS.
+   * when compiling for iOS or tvOS
```
Comment (jonsimantov) [inline firestore/src/include/firebase/firestore/settings.h:171]: Missing period

File: messaging/src/include/firebase/messaging.h [hunk 3]
```diff
@@ -170,7 +170,7 @@ struct Notification {
 
   /// Indicates the key to the title string for localization.
   ///
-  /// On iOS, this corresponds to "title-loc-key" in APNS payload.
+  /// On iOS amd tvOS, this corresponds to "title-loc-key" in APNS payload.
```
Comment (jonsimantov) [inline messaging/src/include/firebase/messaging.h:173]: Typo

**Review by jonsimantov — APPROVED** [review]
Approved with a few typo nits, please fix

File: auth/src/include/firebase/auth.h (hunk 1 above)
Comment (DellaBitta) [reply auth/src/include/firebase/auth.h:177]: :+1:

File: firestore/src/include/firebase/firestore/settings.h (hunk 2 above)
Comment (DellaBitta) [reply firestore/src/include/firebase/firestore/settings.h:171]: Done.

File: messaging/src/include/firebase/messaging.h (hunk 3 above)
Comment (DellaBitta) [reply messaging/src/include/firebase/messaging.h:173]: 👍

**Review by jonsimantov — APPROVED** [review]

**Pull request merged**